### PR TITLE
fix backgroundColor to be pillDefault

### LINKED ISSUE #222: PillStatusBox White Background
The background color should be transparent and not white

### PR DESCRIPTION
**Pull request by DasProffi**
closes #222

### CODE REVIEW

File: apps/tasks/lib/components/task_status_pill_box.dart [hunk 1]
```diff
@@ -40,7 +40,6 @@ class TaskStatusPillBox extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    Color backgroundColor = dividerColor ?? Colors.white;
```
Comment (use-to) [inline apps/tasks/lib/components/task_status_pill_box.dart:43]: Provide `Colors.transparent` as default value for `dividerColor`.

**Review by use-to — CHANGES_REQUESTED** [review]

File: apps/tasks/lib/components/task_status_pill_box.dart (hunk 1 above)
Comment (DasProffi) [reply apps/tasks/lib/components/task_status_pill_box.dart:43]: There is no default value here anymore, instead the default is given by the underlying `TaskStatusPill`, do you want to also set a default in this class?

**Review by florian-sabonchi — APPROVED** [review]

File: apps/tasks/lib/components/task_status_pill_box.dart (hunk 1 above)
Comment (use-to) [reply apps/tasks/lib/components/task_status_pill_box.dart:43]: The real question is do we need a background color?
We could use transparency directly?! I don't see any szenario where we have to set the background color manually? Do you see one?

Comment (DasProffi) [reply apps/tasks/lib/components/task_status_pill_box.dart:43]: When you only want to color the diagonals, then the background color is relevant. To change the diagonal color you would need to add a colored container in the background, which has to match the width and the border radius of the pillbox, which would be require much effort, therefore the attribute.
But we don't use it yet. At the current state it behaves as you want it to by being transparent and therefore I do not see a reason to remove this additional attribute

Comment (use-to) [reply apps/tasks/lib/components/task_status_pill_box.dart:43]: Allright. For the next time we could save the more effort. I am quite sure that we will never touch this attribute.

**Review by use-to — APPROVED** [review]

**Pull request merged**